### DISCUSSION TITLE
The staging server is currently down, and I need to verify the functionality of all modules that I have deployed on my server by comparing to your staging server. can you check please your staging server that is currently having some issues.

### DISCUSSION THREAD

**Discussion by Amitsinghongraph** [egovernments/Digit-Core#279] [Local Governance]
https://staging.digit.org/digit-ui/citizen/error

**Comment by Harish-egov** [comment]
Hi @Amitsinghongraph  Staging URL is working fine now. Because of the db issue url was down.

**Comment by Amitsinghongraph** [comment]
Hi Harish,
I checked just now, and the staging URL is still not working. when I open
the URL  https://staging.digit.org/digit-ui/citizen, it redirects me to the
URL https://staging.digit.org/digit-ui/citizen/error.

regards
amit singh chauhan

**Comment by Harish-egov** [comment]
HI @Amitsinghongraph Please delete the website cookies and then try. It'll work
